demo/src/androidTest/java/com/basgeekball/awesomevalidation/demo/DemoActivityTest.java

```diff
@@ -46,6 +46,7 @@ public void testValidatePass() {
         enterText(R.id.edt_zipcode, "53201");
         enterText(R.id.edt_year, "1982");
         enterText(R.id.edt_height, "1.93");
+        enterText(R.id.edt_birth, "23/09/1994");
```

Comment:
Code is fine, but the complete example I gave here is based on the data of my favourite NBA player Dwyane Wade.  It would be great if you could change the birth to his birthday - January 17, 1982.  Thank you so much for my nonsense reason :P

Reply:
Cool (: